components/InfoPost.js

```diff
@@ -3,22 +3,33 @@ import { parse, format } from 'date-fns'
 import styled from 'styled-components'
 
 import Heart from './Heart'
+import Avatar from './Avatar'
 
-const Avatar = styled.img`
-  width:100%;
-  object-fit:cover;
-  border-radius:50%;
-  border:3px solid ${(props) => props.theme.color.primary};
-  box-sizing: border-box;
-`
+import useWidth from '../hooks/useWidth'
+
+// const Avatar = styled.img`
```

Comment:
You should remove the comments before the Pull Request when you replace it by a Component and You will not use them anymore.